core/modules/layout/plugins/renderers/layout_renderer_editor.inc

```diff
@@ -103,17 +103,18 @@ class LayoutRendererEditor extends LayoutRendererStandard {
    * {@inheritdoc}
    */
   function renderBlock($block) {
+    $aria_label = $this->getAriaLabel($block);
     $buttons = $this->getBlockLinks($block);
     $content = $block->getAdminPreview();
 
-    $output = '<div tabindex="0" class="layout-editor-block clearfix" id="layout-editor-block-' . $block->uuid . '" data-block-id="' . $block->uuid . '">';
+    $output = '<div tabindex="0" aria-label="' . $aria_label . '" class="layout-editor-block clearfix" id="layout-editor-block-' . $block->uuid . '" data-block-id="' . $block->uuid . '">';
```

Comment:
@docwilmot I wonder if it might make sense (for readability) to create an array of attributes and run it through backdrop_attributes() then.

$attributes = array(
  'tabindex' => '1',
  'class' => array(
    ...
  ),
  ...
);

$output = '<div ' . backdrop_attributes($attributes) . '>';
```
That's really only for readability - I find that looong line hard to read. :slightly_smiling_face: 
That function would also do sanitization (with check_plain, though).

Reply:
> create an array of attributes

This makes sense, I agree.
>That function would also do sanitization (with check_plain, though).

I thought since `getAriaLabel()` runs everything through `strip_tags()`, and  `$block->getAdminTitle()` already runs its output through `check_plain()` we'd be OK with that?

Reply:
@docwilmot whatever you choose, it's only a suggestion. :smile:

Reply:
Ahh I misunderstood. You mean `backdrop_attributes()` does sanitization. I thought you were saying that `getAriaLabel()` needs sanitization. Cool, even better.

Reply:
Done